epoxy-adapter/src/test/java/com/airbnb/epoxy/EpoxyVisibilityTrackerTest.kt

```diff
@@ -687,6 +882,7 @@ class EpoxyVisibilityTrackerTest {
     fun setup() {
         Robolectric.setupActivity(Activity::class.java).apply {
             setContentView(EpoxyRecyclerView(this).apply {
+                epoxyVisibilityTracker.setPartialImpressionThresholdPercentage(50)
```

Comment:
That's nice to update all the tests with a `partialImpressionThresholdPercentage` value !

I would have let the existing tests as "it is" and added more test to validate the different `partialImpressionThresholdPercentage` scenarios.

Reply:
Yeah that's why I asked about the test matrix. If necessary, I can write some code to dupe the cases once with and once without partial imps.

> As for tests, since setting the threshold will add a new dimension to the test matrix, I was not sure if I should adjust the whole testsuite to cope for this. Basically all of
> testDataAttachedToRecyclerView, testScrollToPosition, testMoveDataDown, testInsertData, testScrollBy, testDeleteData, testMoveDataUp
> could be tested once with a threshold set, and once without.